.gitignore

```diff
@@ -1,2 +1,5 @@
 # Godot 4+ specific ignores
 .godot/
+
+# Art GIMP files
+_assets
```

Comment:
goofy ahh line not in the local gitignore and muddying the git commit history smh my head

Reply:
you don't need to change this btw